ccpp/data/GFS_typedefs.F90

```diff
@@ -1161,6 +1166,11 @@ module GFS_typedefs
                                               ! multiple patterns. It wasn't fully coded (and wouldn't have worked
                                               ! with nlndp>1, so I just dropped it). If we want to code it properly,
                                               ! we'd need to make this dim(6,5).
+    logical              :: do_spp
+    integer              :: n_var_spp
+    character(len=3)     :: spp_var_list(6)  ! dimension here must match  n_var_max_spp in  stochy_nml_def
```

Comment:
I would recommend to NOT hardcode the length of the `spp_var_list` array here, but follow the logic for `lndp_var_list`. As far as I can see, both `spp_var_list` and `lndp_var_list` are handled in the same way: allocated in GFS_typedefs, but the namelist read happens in the stochastic_physics code. Same for spp_prt_list.

Reply:
Fixed.